content/01.abstract.md

```diff
@@ -1,14 +1,13 @@
-Using degree-preserving network permutations, we define an "edge prior" to quantify the probability that two nodes are connected based only on their degree.
-Applying the edge prior to 20 different biomedical networks (16 bipartite, 3 undirected, 1 directed),
-we demonstrate that node degree information alone is capable of remarkable performance, with AUROCs frequently exceeding 85%.
-The predictive performance of degree diminishes on testing networks that were generated using distinct methods and hence show little degree correlation with the training network.
-Accordingly, we suggest using the edge prior as baseline for edge prediction tasks.
-Performance exceeding the baseline is attributable to factors other than degree alone and is often only a small portion of overall performance.
-We provide an open source Python package, named [`xswap`](https://github.com/hetio/xswap) after the permutation algorithm, that allows users to permute their own networks and compute edge priors.
-Our implementation includes an analytic approximation of the edge prior for efficiency and provides a high-level API to accomodate a variety of network and edge types.
+Important tasks in biomedical discovery such as predicting gene functions, gene-disease associations, and drug repurposing opportunities are often framed as network edge prediction.
+The number of edges connecting to a node, termed degree, can vary greatly across nodes in real biomedical networks.
+If degree strongly influences edge prediction, then imbalance or bias in the distribution of degrees could lead to nonspecific or misleading predictions.
+We introduce a network permutation framework to quantify the effects of node degree on edge prediction.
+Our framwork decomposes performance into the proportion attributable to degree and the proportion attributable to edge-dependent inference.
+Additionally, our permutation-derived edge prior quantifies the probability of a connection based only on node degree.
+This feature shows excellent discrimination and calibration for 20 biomedical networks (16 bipartite, 3 undirected, 1 directed), with AUROCs frequently exceeding 0.85.
+Feature performance exceeding the edge prior is attributable to factors other than degree and is often only a small portion of overall performance.
+In a comparison involving permutated networks and the permutation-derived edge prior, we demonstrate that degree's predictive performance diminishes when the networks used for training and testing were generated using distinct methods and hence have large differences in degree distribution.
+Accordingly, we suggest using the edge prior as a baseline for edge prediction and degree-preserving permutation to investigate a feature's ability to capture degree and non-degree information.
+We released our methods as an open-source Python package on GitHub (https://github.com/hetio/xswap/) and the Python Package Index (https://pypi.org/project/xswap/).
```

Comment:
I'm not sure it really makes sense to put both links. If you go to one, the other is clearly listed. I'd put "the Python Package Index" in the manuscript only not abstract.

Reply:
I agree that you could use just one URL.